app/experimenter/visualization/api/v3/views.py

```diff
@@ -36,6 +38,60 @@ class Statistic:
     COUNT = "count"
 
 
+class JetstreamDataPoint(BaseModel):
+    lower: int = None
+    upper: int = None
+    point: int = None
+    metric: str = None
+    branch: str = None
+    statistic: str = None
+    window_index: str = None
+    comparison: str = None
+
+
+class JetstreamData(BaseModel):
+    __root__: List[JetstreamDataPoint] = []
+
+    def __iter__(self):
+        return iter(self.__root__)
+
+    def append(self, item):
+        self.__root__.append(item)
+
+    def extend(self, item):
+        self.__root__.extend(item)
+
+    def dict(self, exclude_none):
+        return [item.dict(exclude_none=exclude_none) for item in self.__root__]
+
+
+class DataPoint(BaseModel):
+    lower: int = None
+    upper: int = None
+    point: int = None
+    window_index: str = None
+    count: int = None
+
+    def set_window_index(self, window_index):
+        self.window_index = window_index
+
+    def has_bounds(self):
+        return self.lower and self.upper
+
+
+class BranchComparisonData(BaseModel):
+    all: List[DataPoint] = []
+    first: DataPoint = DataPoint()
+
+
+class MetricData(BaseModel):
+    absolute: BranchComparisonData
+    difference: BranchComparisonData
+    relative_uplift: BranchComparisonData
+    significance: dict
```

Comment:
Can you define this in a little more detail?  Like do you know the keys up front?  If you know the keys you can use [TypedDict](https://www.python.org/dev/peps/pep-0589/)

Reply:
ah yes. missed doing this one